neural_style.py

```diff
@@ -487,14 +686,50 @@ def get_init_image(init_type, content_img, style_imgs, frame=None):
   elif init_type == 'random':
     init_img = get_noise_image(args.noise_ratio, content_img)
     return init_img
+  # only for video frames
+  elif init_type == 'prev':
+    init_img = get_prev_frame(frame)
+    return init_img
+  elif init_type == 'prev_warped':
+    init_img = get_prev_warped_frame(frame)
+    return init_img
 
+def get_content_frame(frame):
+  fn = args.content_frame_frmt.format(str(frame).zfill(4))
+  path = os.path.join(args.video_input_dir, fn)
+  img = read_image(path)
+  return img
 
-def get_content_image():
-  img = os.path.join(args.content_img_dir, args.content_img)
+def get_content_image(content_img):
+  path = os.path.join(args.content_img_dir, content_img)
+   # bgr image
+  img = cv2.imread(path, cv2.IMREAD_COLOR)
+  check_image(img, path)
+  img = img.astype(np.float32)
+  h, w, d = img.shape
+  mx = args.max_size
+  # resize if > max size
+  if h > w and h > mx:
+    w = (float(mx) / float(h)) * w
+    img = cv2.resize(img, dsize=(int(w), mx), interpolation=cv2.INTER_AREA)
+  if w > mx:
+    h = (float(mx) / float(w)) * h
+    img = cv2.resize(img, dsize=(mx, int(h)), interpolation=cv2.INTER_AREA)
+  img = preprocess(img)
   return img
 
-def get_style_images():
-  style_imgs = os.path.join(args.style_imgs_dir,args.style_imgs[0])
```

Comment:
Here I see that you are returning style_imgs which is just a path string. though in the original version of the function , the function is returning an array of "IMAGES".so in order to avoid conflicts for those functions which are supposed to call this function(init_img, stylize) we need to make sure the data type of the object we return is still the same so even though we have one image, we should return something like: 
"return [cv2.imread(path, cv2.IMREAD_COLOR)]" 
which is still an array containing our only style image.